Test/e2e/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  api:
+    build:
+      context: ../../App/Server
+      dockerfile: Dockerfile.e2e
```

Comment:
Again, just use the existing Dockerfile and align what's in this docker-compose with that's in Test/Postman for the api portion. Make the port 5000 and update the rest of this file to match that too.